docs/python_docs/python/tutorials/packages/gluon/custom_layer.md

```diff
@@ -0,0 +1,127 @@
+<!--- Licensed to the Apache Software Foundation (ASF) under one -->
+<!--- or more contributor license agreements.  See the NOTICE file -->
```

Comment:
this tutorial is already there: https://mxnet.apache.org/api/python/docs/tutorials/extend/custom_layer.html